src/components/breakpoints/index.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import Responsive from 'react-responsive'
+import { sizes } from 'SRC/core/theme/breakpoints'
+
+export const Desktop = props => <Responsive {...props} minDeviceWidth={sizes.desktop} />;
```

Comment:
What are your thoughts on using CSS instead of the `Responsive` components from `react-responsive`? These components don't work when SSRed.

We can use styled components for this
```js
import { css } from 'styled-components'

export const mobile = inner => css`
  @media (max-width: ${800 / 16}em) {
    ${inner}
  }
`

export const phone = inner => css`
  @media (max-width: ${650 / 16}em) {
    ${inner}
  }
`
```

And then to use
```js
import { mobile } from './media.js` // above file

const Container = styled.div`
  display: none;
  ${mobile(css`
    display: block;
  `)}
```